usr/src/lib/libm_aarch64/src/mapfile-vers

```diff
@@ -171,8 +171,8 @@ global:
 	ctanh;
 	ctanhf;
 #	ctanhl;
-	drem;
-	dremf;
+#	drem;
```

Comment:
Can you remove these rather than comment them out?
I know it sounds weird, and it's also not clear, but the commented symbols are (meant to be, at least) things that are in the i386 and sparc libm, but not this one.  And this is the opposite :)

Reply:
Makes sense. I've removed them now.